docs/apache-airflow/installation/index.rst

```diff
@@ -0,0 +1,335 @@
+files are managed by Apache Airflow release managers to make sure that you can repeatably install Airflow from PyPI with all Providers and
+required dependencies.
+
+You can also verify integrity and provenance of the packages of the packages downloaded from PyPI as described at the installation
+page, but software you download from PyPI is pre-built for you so that you can install it without building.
+
+**Intended users**
+
+Users who are familiar with installing and configuring Python applications, managing Python
+environments, dependencies and running software with their custom deployment mechanisms.
+
+**What are you expected to handle**
+
+You are expected to install Airflow - all components of it - on your own.
+You should develop and handle the deployment for all components of Airflow.
+
+You are responsible for setting up database, creating and managing database schema with ``airflow db`` commands,
+automated startup and recovery, maintenance, cleanup and upgrades of Airflow and Airflow Providers.
+
+**What Apache Airflow Community provides for that method**
+
+You have `instructions <https://airflow.apache.org/docs/apache-airflow/stable/installation.html>`__
+on how to install the software but due to various environments and tools you might want to use, you might expect that there will be
+problems which are specific to your deployment and environment you will have to diagnose and solve.
+
+**Where to ask for help**
+
+The ``#troubleshooting`` channel on Airflow Slack for quick general
+troubleshooting questions. The `GitHub discussions <https://github.com/apache/airflow/discussions>`__
+if you look for longer discussion and have more information to share.
+
+If you can provide description of a reproducible problem with Airflow software, you can open
+issue at `GitHub issues <https://github.com/apache/airflow/issues>`__
+
+
+Using Production Docker Images
+''''''''''''''''''''''''''''''
+
+More details: :doc:`docker-stack:index`
+
+**When this option works best**
+
+This installation method is useful when you are familiar with Container/Docker stack. It provides a capability of
+running Airflow components in isolation from other software running on the same physical or virtual machines with easy
+maintenance of dependencies.
+
+The images are build by Apache Airflow release managers and they use officially released packages from PyPI and official constraint files
+- same that are used for installing airflow from PyPI.
+
+**Intended users**
+
+Users who are familiar with Containers and Docker stack and understand how to build their own container images.
+
+Users who understand how to install providers and dependencies from PyPI with constraints if they want to extend or customize the image.
+
+Users who now how to create deployments using Docker by linking together multiple docker containers and maintaining such deployments.
```

Comment:
What do you think to mention explicitly that this method causes the least problems with installing Airflow?

Reply:
I deliberately wanted to avoid an `assesment` which method is better or worse. I think first of all we can be biased becaue we have certain experience, and secondly - more importantly, this page is for very different people with diverse backgrounds, experience, expectations. 

Something that might be "easiest" for me might be not "easiest" for someone who does not know anything about docker, containers and Kubernetes. It's not for us to judge,  the whole idea of that page is to give the users an objective comparison of what is expected from them if they choose certain path, what experience is needed, and what help and resources they will get. Then, those different users, based on their own backgrounds, habits. expectations, experience, use cases, might make their own choice. 

I think it would very biased to tell that one option is "better" than "other" if we do not know who is reading that document